receiver/collectdreceiver/README.md

```diff
@@ -32,7 +32,7 @@ and, `k` and `v` as the respective label values.
 
 The following settings are required:
 
-- `endpoint` (default = `localhost:8081`): Address to reach the desired Docker daemon.
+- `endpoint` (default = `localhost:8081`): Endpoint exposed by this receiver to send data.
 
```

Comment:
Need to mention that other confighttp.HTTPServerSettings are supported as well

Reply:
Is there a good way to do that? looking at splunkhecreceiver, it's just as terse.

Reply:
ok I like what awsfirehosereceiver does:
> The configuration includes the Opentelemetry collector's server [confighttp](https://github.com/open-telemetry/opentelemetry-collector/tree/main/config/confighttp#server-configuration),
which allows for a variety of settings. Only the most relevant ones will be discussed here, but all are available.